### ISSUE TITLE
Questions about training options.

### ISSUE THREAD

**Issue by liren2515** [nkolot/SPIN#52]
Hi Nikos,

When I check your Supplementary Material, I find the lr is 3e-5, and the maximum number of optimization iterations is 50. But in your code, lr=5e-5, num_smplify_iters=100. So which of them are used in your training?

And you also mentioned "The model with limited access to 3D ground truth (“paired”) was initialized with a model pretrained on Human3.6M [4] using full 3D pose and shape ground truth. Pretraining in this case was useful, such that the model provides better initial 3D shape estimates for the iterative ﬁtting." How do you configure this pretraining? Do you use the same loss and how many iterations does it need?

Another question is, the openpose detections are only used by simplify and will not be considered when you compute the 2D keypoints loss for the network, right?

**Comment by ikvision** [comment]
Concerning the usage of open-pose
https://github.com/nkolot/SPIN/issues/43#issue-549333131 presents a matching issue with open-pose relative to ground truth. It might be that some that a rejection mechanism might reduce this effect.
For my experiments, I got better results when not using open_pose in the 2d_keypoint_loss, only to generate pseudo_ground_truth using simplify. Those pseudo_ground_truth have a rejection mechanism.

**Comment by frankjappa95** [comment]
@ikvision Hi ikvision :) I saw your comment, and you mean that you used OpenPose 2D key point loss only to generate pseudo ground truth?? Since the link(#43 )reveals the improperness of using OpenPose for fitting with key points.... just for the curiosity!
